pkg/ipam/allocator.go

```diff
@@ -135,6 +112,25 @@ func MakeRange(r *lbv1.Range) (*allocator.Range, error) {
 	}, nil
 }
 
+func parseIP(ipStr string, ipNet *net.IPNet, defaultIP net.IP) (net.IP, error) {
+	if ipStr == "" {
+		return defaultIP, nil
+	}
+
+	ip := net.ParseIP(ipStr)
```

Comment:
The variable name `ip` is conflict with the package `github.com/containernetworking/plugins/pkg/ip`

Reply:
I have given it an alias `cnip`.